src/components/SignUp/EditForm.js

```diff
@@ -57,6 +57,7 @@ class EditForm extends Component {
           style={[styles.text, styles.input]}
           defaultValue={this.state.defaultValue}
           underlineColorAndroid="rgba(255, 255, 255, 0)"
```

Comment:
Because I can use it all at once. ^^;
I'll reflect your comment in my next branch!
Thanks!